css/CSS2/floats/computed-float-position-absolute.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<link rel="help" href="https://drafts.csswg.org/css2/visuren.html" />
+<title>The computed value of float with absolute positioning when there is no box should be "none"</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+
+div[id] {
+	position: absolute;
+	float: left;
+}
+
+</style>
+
+<div id="test1" style="display: none"></div>
+<div id="test2" style="display: contents"></div>
+<div style="display: none"><div id="test3"></div></div>
+
+<script>
+var test_description = document.title;
```

Comment:
No need for this, it defaults to `document.title` unless I'm sorely mistaken.

Reply:
Oh, didn't know that. 
In such case, yeah, could be removed.